package.json

```diff
@@ -1,21 +1,33 @@
 {
-  "name": "ratings-and-reviews",
-  "version": "1.0",
-  "description": "Ratings and Reviews sections of a mock ebay",
-  "author": "Napoleon De La Peña",
-  "license": "",
-  "engines": {
-    "node": ">=6.13.0"
+  "name": "napoleon-service",
+  "version": "1.0.0",
+  "description": "> Project description",
+  "main": "./server/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server",
```

Comment:
I would recommending alphabetizing your scripts.

Reply:
This has been adjusted.